### PR TITLE
Release 10.0.11

### PR DESCRIPTION
**Pull request by smalluban**
460fb138176b9363bcedcfaacfd86303821f8d69 Minor fixes for Firefox/Ghostery Browser  (#1173)
ba24ed214c1fc9d8b92ed4be29bc854c322a636e Fix errors that can happen if tabs are closed. Or convert them to warnings.
c100080d05abbe838aed53c8c0781f1b610d2ae1 Fix Youtube adblocking
12d061e3393843d29aaf9f0bac2b2dcd25002b4d Update category icons
e827488df410f15862d1d902c9273aeb2c143c6f Fix Chrome pin instruction image (#1172)

**TestFlight: 10.0.11 (104)**

### CODE REVIEW

**Review by GRadziejewski — DISMISSED** [review]

**Comment by GRadziejewski** [comment]
Tests are finished.

**Comment by smalluban** [comment]
@GRadziejewski Thanks for testing. As there is a need for updating filter engines, please make another round of sanity checks of build 105 tomorrow and re-approve.

File: extension-manifest-v3/scripts/xcode-version.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Open ./xcode/Ghostery.xcodeproj/project.pbxproj and find the line that looks like this:
```
Comment (chrmod) [inline extension-manifest-v3/scripts/xcode-version.js:1]: not sure if this still works but there are tools that are meant to do just this. Please check https://developer.apple.com/library/archive/qa/qa1827/_index.html

**Review by chrmod — APPROVED** [review]

**Comment by GRadziejewski** [comment]
# Iteration nr 2.
Version 105.

## Smoke tests on desktop
- [ ] Browser: Safari 16.3 (18614.4.6.1.6)
OS: MacOS Ventura 13.2.1


## Smoke tests on mobile
- [ ] Browser: Safari
OS: iOS 16.4.1


## Checked PRs / bugfixes
- 

## New issues
-

**Comment by GRadziejewski** [comment]
Tests are finished.

**Pull request merged**